akka-testkit/src/main/scala/akka/testkit/javadsl/TestKit.scala

```diff
@@ -615,6 +615,7 @@ class TestKit(system: ActorSystem) {
    * one of the given classes. Wait time is bounded by the given duration,
    * with an AssertionFailure being thrown in case of timeout.
    */
+  @varargs
```

Comment:
Doesn't this cause a binary incompatibility?

Reply:
It's in the testkit so that is ok, we don't keep bincomp here

Reply:
ah :+1: that makes sense of course